datahub-web-react/src/app/useGetAuthenticatedUser.tsx

```diff
@@ -6,7 +6,7 @@ import { useGetUserQuery } from '../graphql/user.generated';
  * Fetch a CorpUser object corresponding to the currently authenticated user.
  */
 export function useGetAuthenticatedUser() {
-    const userUrn = Cookies.get(CLIENT_AUTH_COOKIE);
+    const userUrn = process.env.JEST_WORKER_ID !== undefined ? 'urn:li:corpuser:1' : Cookies.get(CLIENT_AUTH_COOKIE);
```

Comment:
let's not include anything related to tests in this method- if you need to mock this it should be mocked in the test file itself.